src/main.rs

```diff
@@ -64,6 +67,9 @@ struct RunArgs {
     /// will not be saved
     #[argh(option)]
     stats_dir: Option<PathBuf>,
+
+    #[argh(switch, description = "accept invalid certificates")]
```

Comment:
Could you please change both of the descriptions to "disable verification of TLS certificates"?

Reply:
Should be good now